src/components/Api.js

```diff
@@ -0,0 +1,99 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _results(res) {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`_checkResponse` 

You check the reponse here